<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab1DB</class>
 <widget class="QWidget" name="Tab1DB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>390</width>
        <height>1309</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="pCBkey1">
           <property name="text">
            <string>사과</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey2">
           <property name="text">
            <string>바나나</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey3">
           <property name="text">
            <string>복숭아</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey4">
           <property name="text">
            <string>멜론</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey5">
           <property name="text">
            <string>포도</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey6">
           <property name="text">
            <string>귤</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey7">
           <property name="text">
            <string>오렌지</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey8">
           <property name="text">
            <string>샤인머스켓</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey9">
           <property name="text">
            <string>딸기</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey10">
           <property name="text">
            <string>수박</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey11">
           <property name="text">
            <string>파</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey12">
           <property name="text">
            <string>양파</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey13">
           <property name="text">
            <string>마늘</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey14">
           <property name="text">
            <string>감자</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey15">
           <property name="text">
            <string>고구마</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey16">
           <property name="text">
            <string>양배추</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey17">
           <property name="text">
            <string>무</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey18">
           <property name="text">
            <string>칸초</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey19">
           <property name="text">
            <string>조리퐁</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey20">
           <property name="text">
            <string>쿠쿠다스</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey21">
           <property name="text">
            <string>초코칩쿠키</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey22">
           <property name="text">
            <string>포카칩</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey23">
           <property name="text">
            <string>치토스</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey24">
           <property name="text">
            <string>뿌셔뿌셔</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey25">
           <property name="text">
            <string>새우깡</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey26">
           <property name="text">
            <string>냄비</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey27">
           <property name="text">
            <string>국자</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey28">
           <property name="text">
            <string>후라이팬</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey29">
           <property name="text">
            <string>콜라</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey30">
           <property name="text">
            <string>사이다</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey31">
           <property name="text">
            <string>포카리</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey32">
           <property name="text">
            <string>델몬트</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey33">
           <property name="text">
            <string>환타</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey34">
           <property name="text">
            <string>파워에이드</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey35">
           <property name="text">
            <string>핸드워시</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey36">
           <property name="text">
            <string>바디워시</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pCBkey37">
           <property name="text">
            <string>샴푸</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pPBappQuit">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pPBClear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pPBStart">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Padauk Book</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Shopping Lists</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
